lib/config.js

```diff
@@ -134,21 +137,24 @@ export default class Config {
 
     this.resolveLoader = {
       root: [
-        path.join(__dirname, '../node_modules'), // local, global
-        path.join(__dirname, '../../../node_modules') // flattened, via npm 3+
+        path.join(opts.root), // the project root
+        path.join(__dirname, '../node_modules'), // roots-mini/node_modules
+        path.join(__dirname, '../../../node_modules') // roots-mini's flattened deps, via npm 3+
       ]
     }
 
     // ignore node_modules and output directory automatically
     opts.ignore.unshift('**/node_modules/**', `${this.output.path}/**`)
 
+    const rootsLoaders = [
+      { test: mmToRe(opts.matchers.css), exclude: opts.ignore.map(mmToRe), loader: 'css-loader!postcss-loader' },
```

Comment:
@jenius this was the real root of our "not a postCss plugin" error. because we had it defined as simply `postcss`, webpack was finding our `roots-mini/node_modules/postcss` (flattened from our internal postcss-* plugins) and running whatever function it exports. apparently if you're explicit, it won't traverse that low.